### ISSUE TITLE
Let's do composites

### ISSUE THREAD

**Issue by adisen99** [navidcy/LaNina-flavours#2]
The movie looks great and gives a really cool animation of the development of ENSO events. I was wondering if we could isolate this for a composite of CP and EP La Nina events.

For e.g. just take the composite of the CP LN events and then look at the animation of the lead and lag composites rather than looking at a long animation.

We could then add this video to the supplementary and have the still images in the main text of the paper. 

This would also help in interpretation of the paper.

**Comment by h-hdm** [comment]
I agree, this is a great idea!! :) Is this possible to generate for each type of La Nina event? @navidcy

**Comment by adisen99** [comment]
Hey @navidcy, in order to work on the composites, I was trying to use the notebook and run the cells but running into errors in the code box -

```python
session = cc.database.create_session()
```


ERROR msg -

```
OperationalError: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```



I was wondering perhaps it is easier if you could save the detrended data as a netcdf file somewhere on NCI and then I could just use that to create the composites.

**Comment by navidcy** [comment]
We've done composites. ;)
